src/components/NextProject/NextProject.vue

```diff
@@ -0,0 +1,130 @@
+<template lang="html">
+    <nuxt-link :to="path" class="next-project">
+        <wp-image :image="image" mode="fullbleed" />
+        <div class="meta">
+            <h4>Next Project</h4>
+            <div class="credits">
+                <split-text class="title" :text="title" />
+                <p
+                    v-for="credit in formattedCredits.slice(1)"
+                    v-if="credits"
+                    :key="credit"
+                    class="credit"
+                    v-html="credit"
+                />
+            </div>
+        </div>
+    </nuxt-link>
+</template>
+
+<script>
+// Helpers
+import { nl2br } from "@/utils/tools";
+
+// Components
+import WpImage from "@/components/global/WpImage";
+import NuxtLink from "@/components/global/NuxtLink";
+import SplitText from "@/components/global/SplitText";
+
+export default {
+    components: {
+        WpImage,
+        NuxtLink,
+        SplitText
+    },
+    props: {
+        image: {
+            type: Object,
+            default: () => {}
+        },
+        title: {
+            type: String,
+            default: ""
+        },
+        credits: {
+            type: String,
+            default: ""
+        },
+        path: {
+            type: String,
+            default: ""
+        }
+    },
+    computed: {
+        formattedCredits() {
+            return nl2br(this.credits).split("<br>");
+        }
+    }
+};
+</script>
+
+<style lang="scss" scoped>
+.next-project {
+    background-color: var(--color-black); //placeholder
```

Comment:
Remove `//placeholder` please. Leave the color as you have it.

Also, in global.scss, can you add `background-color: #000000` to the body, just makes sense for a site where the background is dark.